ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml

```diff
@@ -14,6 +14,7 @@ import utils 1.0
 Item {
     id: root
 
+    property var rootStore
```

Comment:
Panels should have no reference to stores (https://hackmd.io/@status-desktop/main/https%3A%2F%2Fhackmd.io%2F%40status-desktop%2FB1eOaf-nd)

Reply:
changed